shared/actions/pinentry.js

```diff
@@ -60,50 +49,38 @@ const onNewPinentry = (_, action) =>
   })
 
 const onSubmit = (_, action) => {
-  const {sessionID, passphrase} = action.payload
-  _respond(sessionID, {passphrase})
+  const {passphrase} = action.payload
+  if (_response) {
+    // $FlowIssue flow is correct in that we need store secret but we weren't sending it before and i don't want to change any existing behavior
+    _response.result({passphrase})
```

Comment:
flow will infer _response correctly now and it correctly points out we're missing a param. i didn't want to break anything so i left it as-is